### PR TITLE
Adding documentation for QNode

### PR DESCRIPTION
**Pull request by MarkDWilliams**

### CODE REVIEW

**Review by ehinderer — CHANGES_REQUESTED** [review]
Typo in the first sentence, should be "...used **to** represent an entity..." Otherwise fine I think.

**Review by vdancik — APPROVED** [review]

File: TranslatorReasonerAPI.yaml [hunk 1]
```diff
@@ -290,7 +290,9 @@ components:
         - edges
     QNode:
       type: object
-      description: A node in the QueryGraph
+      description: A node in the QueryGraph used represent an entity in a
+        query. If a CURIE is not specified, any nodes matching the type
+        of the QNode will be returned is the Results.
```
Comment (rtroper) [inline TranslatorReasonerAPI.yaml:295]: Another little typo: Did you mean to say "...will be returned [in] the Results"?

File: TranslatorReasonerAPI.yaml [hunk 2]
```diff
@@ -290,7 +290,9 @@ components:
         - edges
     QNode:
       type: object
-      description: A node in the QueryGraph
+      description: A node in the QueryGraph used to represent an entity in a
+        query. If a CURIE is not specified, any nodes matching the type
```
Comment (patrickkwang) [inline TranslatorReasonerAPI.yaml:294]:
```suggestion
        query. If a CURIE is not specified, any nodes matching the category
```

**Review by patrickkwang — CHANGES_REQUESTED** [review]

**Review by patrickkwang — APPROVED** [review]

**Comment by MarkDWilliams** [comment]
I believe the typos and suggestions have all been corrected/made.

**Review by edeutsch — APPROVED** [review]

**Pull request merged**